lnrpc/rpc.proto

```diff
@@ -1657,6 +1657,19 @@ message QueryRoutesRequest {
     the optimal route.
     */
     bool use_mission_control = 9;
+
+    /**
+    A list of directed node pairs that will be ignored during path finding.
+    */
+    repeated NodePair ignored_pairs = 10;
```

Comment:
somewhat unfortunate that this requires 33*2/8 = 8.25x more bandwidth to transmit (assuming single channel), but probably okay unless the set is huge. 

are there any plans to make this streaming so the whole list isn't transmitted on each request?

Reply:
I don't have those plans, but I find it difficult to foresee if this becomes a problem, because it depends on the actual use of the functionality.

Reply:
definitely, i just mean that, atm, this would require a quadratic amount of bandwidth/serialization to do an iterative blacklist. i don't think it'd be to hard to switch to a streaming mode later if we find it an issue